plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -16,46 +16,64 @@
 short_description: Manage External EPG in schema of sites
 description:
 - Manage External EPG in schema of sites on Cisco ACI Multi-Site.
-- This module can only be used on versions of MSO that are 3.3 or greater.
+- This module can only be used on versions of MSO that are 3.3 or greater
```

Comment:
Why are we removing all the dots behind sentences?

Reply:
not removing as discussed